sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -185,121 +185,11 @@ The following section provides several code snippets illustrating common pattern
 ### Recognize Content
 Recognize text, tables, and selection marks like radio buttons and check boxes data, along with their bounding box coordinates, from documents.
 
-```C# Snippet:FormRecognizerSampleRecognizeContentFromUri
```

Comment:
Will you be adding samples in a later PR?

Reply:
Yes. When I work on the Readme, I will add the real samples here